content/en/seachat/seachat-manual/06-workspace/01-workspace-management.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Workspace Management"
+description: "SeaChat Workspace Management"
+lead: ""
+date: 2020-10-06T08:48:45+00:00
+lastmod: 2024-06-27T08:48:45+00:00
+weight: 61
+draft: false
+images: []
+aliases:
+  - /en/seachat/seachat-manual/workspace/01-workspace-management
+  - /seachat/seachat-manual/workspace/01-workspace-management
+---
+
+## Agents
+
+<div id="additional-setting-ui" style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-content: center">
+    <a href="/images/seachat/en/workspace/01-workspace-management/agents.png" target="_blank">
+    <img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/workspace/01-workspace-management/agents.png" alt="">
+    </a>
+</div>
+    <p style="margin-top: 20px; font-size: 15px">Agent Management</p>
+</div>
+
+This is where you can manage all your AI agents. Each row represents an agent and its name, email, and status. You can also see the number of unread messages. Find the URL to the agent conversation, launch the conversation directly, or add new agents here.
+
+## Members
+
+<div id="additional-setting-ui" style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-content: center">
+    <a href="/images/seachat/en/seachat-manual/workspace/01-workspace-management/members.png" target="_blank">
+    <img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/workspace/01-workspace-management/members.png" alt="">
+    </a>
+</div>
+    <p style="margin-top: 20px; font-size: 15px">Members Management</p>
+</div>
+
+There are different types of members. Depending on the type of membership, members will have different access and permissions. If you are the owner of a workspace, you can add new members to your workspace and assign different roles to them. Here are some of the roles you can assign to a member:
+
+1. **Admin**: Admins have full access to the workspace and can manage all the settings. They can edit the AI agents' settings, manage the knowledge base, access the agent conversation, and take over conversations as human agents.
+2. **AI Agent Editor**: AI Agent Editors can edit the AI agents' settings, manage the knowledge base, and access the agent conversation and testing.
+3. **Human Agent**: Human Agents can access the agent conversation and take over conversations as human agents.
+
+### Agent Assignment
```

Comment:
is this Agent Assignment or Member?

![image](https://github.com/seasalt-ai/wiki/assets/543547/eade1320-7599-4542-947b-20f0858363a0)